content/team.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Team
+layout: page
+---
+
+# PyCon India 2021 Team
+
+The conference wouldn't have concluded on a high note much less even started planning without the constant efforts of countless volunteers which were part of this year's workgroups. The following is a list of individuals that came together to work for 10+ months in order to make the online conference this year a great experience for everyone attending.
+
+- Abhishek Mishra
+- Abhishek Upperwal
+- Akshay Gaikwad
+- Anirudh Menon
+- Aravind Sekar
+- Arun
+- Bharat
+- Bhavin Gandhi
+- Chandan Kumar
+- Darshna Das
+- Divya
+- Duranjai Malik
+- Farhaan B
+- Gajendra
+- Garvit Kulshreshtha
+- Girish
+- Hari priya
+- Harsh Mittal
+- Jatin Goel
+- Kalyan Prasad
+- Kumar Anirudha
+- Laisha Wadhwa
+- Megha
+- Minal
+- Nabarun Pal
+- Nikhil Maan
+- Pradhvan Bisht
+- Pravin Patil
+- Priyanshu Kumar
+- Rahul B
+- Rahul Kumaresan
+- Rayan Das
+- Ritesh
+- Rohan Samanta
+- Sanchit
+- Sandeep Choudhary
+- Saptak
+- Sayan
+- Shashank Kumar
+- Shivang Dubey
+- Subho
+- Suchita Pallavi
+- Sukanya Mandal
+- Tiyasha Banerjee
+- Vaibhav Mishra
+- Vipul Gupta
+- Vishal
```

Comment:
@realslimshanky did we have another Vishal than Vishal Vijayraghavan (who handled stage 4)?

Probably these two are just same.

Reply:
@bhavin192 they are same.